Cargo.toml

```diff
@@ -35,6 +35,7 @@ log = { version = "0.4", default-features = false }
 qlog = { version = "0.13", default-features = false }
 quinn-udp = { version = "0.5.6", default-features = false, features = ["direct-log", "fast-apple-datapath"] }
 regex = { version = "1.9", default-features = false, features = ["unicode-perl"] }
+rustc-hash = { version = "2.1", default-features = false, features = [ "std" ]}
```

Comment:
Note that mozilla-central uses `1.1.0`.

https://searchfox.org/mozilla-central/rev/e85232b4b28ecc970240d39203e417d1c320623c/Cargo.lock#5620-5624